share/man/man7/networking.7

```diff
@@ -0,0 +1,122 @@
+.\"-
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.\" Copyright (c) 2023 Alexander Ziaee
+.\"
```

Comment:
If you want, you can omit the actual text of the license now.

Reply:
Thank you, added in latest commit